app/src/main/java/com/framgia/englishconversation/screen/audiodetail/AudioDetailViewModel.java

```diff
@@ -50,15 +53,39 @@
     private AudioDetailContract.Presenter mPresenter;
     private FragmentManager mManager;
     private CommentFragment mFragment;
+    private UserModel mTimelineUser;
+    private OnTimelineItemTouchListener mTouchListener =
+            new OnTimelineItemTouchListener<TimelineModel>() {
+                @Override
+                public void onItemTimelineClick(TimelineModel item) {
+                    //TODO
+                }
+
+                /**
+                 * @param item truyền vào khi người dùng click vào layout header item
+                 * check điều kiện nếu userModel từ profile gừi sang trùng với user người tạo
+                 * post thì không điều hướng sang activity profile mới
+                 */
+                @Override
+                public void onItemUserNameClick(TimelineModel item) {
+                    if (mTimelineUser != null && mTimelineUser.getId()
+                            .equals(item.getCreatedUser().getId())) {
+                        return;
+                    }
```

Comment:
ERROR: 'if rcurly' have incorrect indentation level 20, expected level should be one of the following: 12, 16.